MIDIchlorians/MIDIchlorians/DataManager.swift

```diff
@@ -195,7 +195,7 @@ class DataManager {
         }
 
         do {
-            try realm?.write { realm?.add(Audio(value: audioFile)) }
+            try realm?.write { realm?.add(Audio(value: [audioFile])) }
```

Comment:
i'll merge this in, you can make your changes in another PR to standardize everything :)